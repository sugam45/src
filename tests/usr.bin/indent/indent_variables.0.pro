/* $NetBSD: indent_variables.0.pro,v 1.1 2021/03/09 20:43:20 rillig Exp $ */
/* $FreeBSD$ */

-l20
-di0
