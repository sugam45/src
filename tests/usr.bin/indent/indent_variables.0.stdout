/* $NetBSD: indent_variables.0.stdout,v 1.1 2021/03/09 20:43:20 rillig Exp $ */
/* $FreeBSD$ */

/*-
 * Demonstrate how variable declarations are broken into several lines when
 * the line length limit is set quite low.
 */

struct s0 a,
   b;
struct s01 a,
    b;
struct s012 a,
     b;
struct s0123 a,
      b;
struct s01234 a,
       b;
struct s012345 a,
        b;
struct s0123456 a,
         b;
struct s01234567 a,
          b;
struct s012345678 a,
           b;
struct s0123456789 a,
            b;
struct s01234567890 a,
             b;
struct s012345678901 a,
              b;
struct s0123456789012 a,
               b;
struct s01234567890123 a,
                b;
